src/content/migrate/5.md

```diff
@@ -156,3 +156,8 @@ Create an [issue](https://github.com/webpack/webpack/issues/new?template=Bug_rep
 
 Please open a Pull Request to help the next person using this guide.
 
+## Changes to internals
+
+The changes to webpack internals like for example: adding types, refactoring code and methods renaming are listed here for anyone interested. But they are not intended as a part of common use-case migration.
```

Comment:
```suggestion
The changes to webpack internals such as: adding types, refactoring code and methods renaming are listed here for anyone interested. But they are not intended as a part of common use-case migration.
```

Reply:
Or `like` can for too

Reply:
Do we need a comma befire such as? I am not sure what you guys thing?

Reply:
like and for example seems a bit redundant to me together conveying the same meaning, we can use either of them or something like `such as`.

Reply:
yes i was saying if we might need to add a comma before `such as`